src/main/java/plugily/projects/buildbattle/arena/states/build/StartingState.java

```diff
@@ -49,22 +51,23 @@ public void handleCall(PluginArena arena) {
     if (arena.getTimer() == 0 || arena.isForceStart()) {
       //getPlotManager().resetPlotsGradually();
       pluginArena.setParticleRefreshScheduler(new ParticleRefreshScheduler(getPlugin()));
+
       if(!pluginArena.getPlotManager().isPlotsCleared()) {
         pluginArena.getPlotManager().resetQueuedPlots();
       }
+
       pluginArena.distributePlots();
       setArenaTimer(getPlugin().getConfig().getInt("Time-Manager." + pluginArena.getArenaType().getPrefix() + ".Voting.Theme"));
+
       pluginArena.setArenaInGameStage(BaseArena.ArenaInGameStage.THEME_VOTING);
+
+      Main plugin = (Main) getPlugin();
+
       for (Player player : arena.getPlayers()) {
-        player.getInventory().setItem(8, getPlugin().getSpecialItemManager().getSpecialItemStack("OPTIONS_MENU"));
+        player.getInventory().setItem(8, plugin.getOptionsRegistry().getMenuItem());
```

Comment:
using pluginArena.getPlugin() to avoid issues and a calling

Reply:
oh didn't saw this method